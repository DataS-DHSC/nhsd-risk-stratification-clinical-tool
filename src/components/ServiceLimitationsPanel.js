import React from 'react';
import { Panel } from 'nhsuk-react-components';

const ServiceLimitationsPanel = () => (
  <Panel
    id="warning-panel"
    className="nhsuk-warning-callout nhsuk-u-margin-top-7 nhsuk-u-margin-bottom-4 nhsuk-u-padding-bottom-8"
  >
    <h3 id="warning-score-heading" className="nhsuk-warning-callout__label">
      Current limitations of this service
    </h3>
    <p>
      The service has been developed for use by clinicians and health care
      professionals. It should be used alongside clinical judgement.
    </p>
    <ul>
      <li id="limitation-underestimate">
        The service may currently underestimate the risk of COVID-19 to patients
        who were shielding during the first peak of the Coronavirus (COVID-19)
        pandemic
      </li>
      <li id="limitation-no-removal">
        Therefore this service should not currently be used to add or remove
        patients from the Shielded Patients List
      </li>
    </ul>
  </Panel>
);

export default ServiceLimitationsPanel;
