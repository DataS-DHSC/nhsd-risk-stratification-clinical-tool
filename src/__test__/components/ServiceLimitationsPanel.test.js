import React from 'react';
import { render } from '@testing-library/react';
import ServiceLimitationsPanel from '../../components/ServiceLimitationsPanel';

const setup = () => render(<ServiceLimitationsPanel />);

describe('ServiceLimitationsPanel', () => {
  it('ServiceLimitationsPanel should render', () => {
    const container = setup();
    expect(container.asFragment()).toMatchSnapshot();
  });
});
