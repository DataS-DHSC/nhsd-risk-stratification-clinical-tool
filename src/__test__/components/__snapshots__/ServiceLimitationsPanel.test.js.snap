// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ServiceLimitationsPanel ServiceLimitationsPanel should render 1`] = `
<DocumentFragment>
  <div
    class="nhsuk-panel nhsuk-warning-callout nhsuk-u-margin-top-7 nhsuk-u-margin-bottom-4 nhsuk-u-padding-bottom-8"
    id="warning-panel"
  >
    <h3
      class="nhsuk-warning-callout__label"
      id="warning-score-heading"
    >
      Current limitations of this service
    </h3>
    <p>
      The service has been developed for use by clinicians and health care professionals. It should be used alongside clinical judgement.
    </p>
    <ul>
      <li
        id="limitation-underestimate"
      >
        The service may currently underestimate the risk of COVID-19 to patients who were shielding during the first peak of the Coronavirus (COVID-19) pandemic
      </li>
      <li
        id="limitation-no-removal"
      >
        Therefore this service should not currently be used to add or remove patients from the Shielded Patients List
      </li>
    </ul>
  </div>
</DocumentFragment>
`;
